Assets/Scripts/DataController.cs

```diff
@@ -38,6 +38,8 @@ public List<Star> AllStars
         get { return allStars; }
         private set { allStars = value; }
     }
+    [SerializeField]
```

Comment:
minor, but only need that attribute on private fields

Reply:
Oops.  My oversight.  Will fix.